octoprint_PrintTimeGenius/printer_config.py

```diff
@@ -134,6 +134,10 @@ def __iadd__(self, new_line):
     mcode = get_code(new_line, "M")
     if not mcode:
       return self  #Save time by quiting early
+    # add T0 when extruder index is ommited to correctly merge the mcodes
+    if mcode == "M221":
+      if get_code(new_line, "T") == "":
+        new_line += " T0"
```

Comment:
Want to add some doctest around line 125?

https://docs.python.org/3/library/doctest.html